smartapps/dianoga/netatmo-connect.src/netatmo-connect.groovy

```diff
@@ -11,19 +11,22 @@ private getVendorTokenPath(){ "https://api.netatmo.com/oauth2/token" }
 private getVendorIcon()		{ "https://s3.amazonaws.com/smartapp-icons/Partner/netamo-icon-1%402x.png" }
 private getClientId() 		{ appSettings.clientId }
 private getClientSecret() 	{ appSettings.clientSecret }
-private getServerUrl() 		{ "https://graph.api.smartthings.com" }
+private getServerUrl() 		{ return "https://graph.api.smartthings.com" }
+private getShardUrl()		{ return getApiServerUrl() }
+private getCallbackUrl()	{ "https://graph.api.smartthings.com/oauth/callback" }
```

Comment:
Might be good to make this configurable so it can more easily be tested in lower environments when that becomes more common place.